crates/rapier3d-meshloader/src/lib.rs

```diff
@@ -48,12 +48,14 @@ pub fn load_from_path(
     let scene = loader.load(path)?;
     for (raw_mesh, _) in scene.meshes.into_iter().zip(scene.materials) {
         let shape = load_from_raw_mesh(&raw_mesh, converter, scale);
-
-        colliders.push(shape.map(|(shape, pose)| LoadedShape {
-            shape,
-            pose,
-            raw_mesh,
-        }));
+        colliders.push(match shape {
+            Ok((shape, pose)) => Ok(LoadedShape {
+                shape,
+                pose,
+                raw_mesh,
+            }),
+            Err(e) => Err(e),
+        });
```

Comment:
Why was that change needed?

Reply:
I probably added a log in between and then forgot, I'll revert it